inc/templates/head-management.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Automatically inject an `irving/head` component to manage the document
+ * <head>.
+ *
+ * @package WP_Irving
+ */
+
+namespace WP_Irving\Templates;
```

Comment:
This should either be a sub-namespace, like `WP_Irving\Templates\Head_Management` or these functions should live in a general `inc/templates/namespace.php` file.